packages/devtools/client/app.vue

```diff
@@ -126,11 +126,11 @@ registerCommands(() => [
     <div
       v-else
       :class="isUtilityView ? 'flex' : sidebarExpanded ? 'grid grid-cols-[250px_1fr]' : 'grid grid-cols-[50px_1fr]'"
-      h-full h-screen of-hidden font-sans bg-base
+      h-full h-screen of-hidden rounded-xl font-sans bg-base
     >
       <SideNav v-show="!isUtilityView" of-x-hidden of-y-auto />
       <NuxtLayout>
-        <NSplitPane storage-key="devtools:split-screen-mode" :min-size="20">
+        <NSplitPane storage-key="devtools:split-screen-mode" :min-size="20" aaa>
```

Comment:
```suggestion
        <NSplitPane storage-key="devtools:split-screen-mode" :min-size="20">
```

Reply:
![PXL_20231202_175220467.jpg](https://github.com/nuxt/devtools/assets/19751938/2c904459-76dd-400e-b97a-3b254bb731c9)

It was him sorry (867cd65)